src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicRename.cs

```diff
@@ -359,7 +359,7 @@ Inherits Attribute
 Import System;
 
 Public Class CustomAttribute
-        Inherits Attribute
+    Inherits Attribute
```

Comment:
@mavasani Please verify that the new behavior in this file is correct and not a regression

Reply:
Seems like completion is changing the formatting for the next line. I believe the previous behavior is more desirable, which seems to be the case behavior in presence of async completion.

Reply:
➡️ Filed #33799